core/src/test/scala/org/apache/spark/shuffle/sort/SortShuffleManagerSuite.scala

```diff
@@ -85,6 +85,14 @@ class SortShuffleManagerSuite extends SparkFunSuite with Matchers {
       mapSideCombine = false
     )))
 
+    // We support serialized shuffle if we do not need to do map-side aggregation
+    assert(canUseSerializedShuffle(shuffleDep(
+      partitioner = new HashPartitioner(2),
+      serializer = kryo,
+      keyOrdering = None,
+      aggregator = Some(mock(classOf[Aggregator[Any, Any, Any]])),
+      mapSideCombine = false
```

Comment:
Under what scenario will ```mapSideCombine``` be ```false```, but an ```aggregator ```  set ?

Reply:
You can see this code: `def groupByKey(partitioner: Partitioner): RDD[(K, Iterable[V])]`